rhel/SPECS/pmm-update.spec

```diff
@@ -40,12 +54,16 @@ cp -pav ./bin/* %{buildroot}%{_bindir}
 install -d %{buildroot}%{_datadir}/%{name}
 cp -pav ./ansible %{buildroot}%{_datadir}/%{name}
 
+cd src/github.com/percona/pmm-update
```

Comment:
why we need to cd into it?

Reply:
we need to do it to build go code, it is standard approach

Reply:
but it it not for build action